app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/KeyCache.java

```diff
@@ -0,0 +1,31 @@
+package gov.hhs.cdc.trustedintermediary.etor;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/** Stores keys from Azure to optimize performance. */
+public class KeyCache {
+
+    private Map<String, String> keys;
+
+    private KeyCache() {
+        // ConcurrentHashMap<>() over Collections.synchronizedMap() due to performance. Synchronized
+        // map, locks the
+        // whole object when reading or writing. Concurrent hashmap locks happen at the bucket
+        // level, leaving the read
+        // function unlocked when writing.
```

Comment:
I think the linter formatted the comment in this way. I can see if it will accept making it more symmetrical, thoughts?

Reply:
Looks better now